programs/odbc-bridge/SchemaAllowedHandler.cpp

```diff
@@ -70,13 +70,19 @@ void SchemaAllowedHandler::handleRequest(HTTPServerRequest & request, HTTPServer
         return;
     }
 
+     bool use_connection_pooling = params.getParsed<bool>("use_connection_pooling", true);
```

Comment:
isn't it more correct to name it "use_connection_pool"?

Reply:
I don't have any strong opinions about this - to me they are more are less similar in meaning.

Reply:
Ok